Makefile

```diff
@@ -32,9 +39,15 @@ ta:
 			  $@
 
 .PHONY: clean
+ifneq ($(wildcard $(TA_DEV_KIT_DIR)/host_include/conf.mk),)
 clean:
 	$(q)$(MAKE) -C host/xtest O=$(out-dir)/xtest q=$(q) $@
 	$(q)$(MAKE) -C ta O=$(out-dir)/ta q=$(q) $@
+else
+clean:
+	$(q)echo "TA_DEV_KIT_DIR is not correctly defined - Remove $(out-dir)"
+	$(q)rm -rf $(out-dir)
```

Comment:
Another question: should I raise an error or not:
* if no error, then this is not dangerous because we may think it has been cleaned, but it has not
* if an error is raised, then, from build, the first `make all-clean` will work, but not the second one

(So long thread for something short)